usr/sbin/registercloudguest

```diff
@@ -214,6 +223,7 @@ if (args.email and not args.reg_code):
 
 time.sleep(int(args.delay_time))
 
+
```

Comment:
? why the additional whitespace?

Reply:
fat fingers, overlooked that change, fixed